protocol.md

```diff
@@ -1213,17 +1213,29 @@ _Response_:
 * result: [`Location`](#location) | [`Location`](#location)[]
 * error: code and message set in case an exception happens during the definition request.
 
-#### Get Symbol Request
+#### Resolve Symbol Request
 
-The get symbol request is sent from the client to the server to resolve the definition location and symbol information for a given text document position.
+The resolve symbol request is sent from the client to the server to resolve the definition location, symbol information, and related symbols for a given text document position.
 
 _Request_
-* method: 'textDocument/symbol'
+* method: 'textDocument/resolveSymbol'
 * params: [`TextDocumentPositionParams`](#textdocumentpositionparams)
 
 _Response_:
-* result: `SymbolInformation` | `SymbolInformation`[]
-* error: code and message set in case an exception happens during the definition request.
+* result: `ResolvedSymbolInformation` | `ResolvedSymbolInformation`[], defined as follows:
```

Comment:
Hm - I guess the only reason it made sense to me is that there is going to be a lot of extra data, and for workspace/symbol the user is doing more of a cursory search, and in the search case the need for ResolvedSymbols seems like less of a thing - one is mainly looking for locations. it would be really duplicative to send back related symbols, since the child nodes would indicate their parents and vice versa. Thoughts?